core/src/main/codegen/templates/Parser.jj

```diff
@@ -7274,6 +7305,7 @@ SqlPostfixOperator PostfixRowOperator() :
 |   < ASSIGNMENT: "ASSIGNMENT" >
 |   < ASYMMETRIC: "ASYMMETRIC" >
 |   < AT: "AT" >
+|   < ATSYM: "@" >
```

Comment:
It doesn't seem like this is used, so it probably should be removed.

Reply:
Done.